binderhub/repoproviders.py

```diff
@@ -111,6 +121,21 @@ def has_higher_quota(self):
                 return True
         return False
 
+    def spec_configuration_override(self):
+        """
+        Return the spec configuration overrides if the given spec has them.
+        """
+        if not isinstance(self.spec_configuration_override, dict):
```

Comment:
I think because you [specified `Dict` for the option](https://github.com/jupyterhub/binderhub/pull/888/files#diff-c5688934f1e6dc3e932b6c84c1bbbd5dR83) we should never get here/have to check this as traitlets checks for us.

Reply:
ah ok - so it'll raise an error if somebody passes a list and not a dict in the helm config?